elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -398,15 +398,20 @@ private SQLQuery toPageTotalSQL(Query query, SQLQuery sql, SQLDialect sqlDialect
                         .map((column) -> column.toSQL(queryReferenceTable))
                         .collect(Collectors.joining(", "));
 
-        String projectionClause = sqlDialect.generateCountDistinctClause(groupByDimensions);
-
-        return SQLQuery.builder()
-                .projectionClause(projectionClause)
+        SQLQuery innerQuery =  SQLQuery.builder()
+                .projectionClause(groupByDimensions)
                 .fromClause(sql.getFromClause())
                 .joinClause(sql.getJoinClause())
                 .whereClause(sql.getWhereClause())
+                .groupByClause(String.format("GROUP BY %s", groupByDimensions))
```

Comment:
Is there any reason we can't simply do:
.groupByClause(sql.getGroupByClause)
and avoid this extra bit of code here?
What if there are no group by dimensions?